### PR TITLE
Jc/add semantic logging to template

### PR DESCRIPTION
**Pull request by JamieCleare2525**
Add semantic logger setup to template for integration with Kibana.

### CODE REVIEW

**Review by saliceti — APPROVED** [review]
👏
(not tested though)

File: templates/semantic_logger.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  # Semantic logging for integration with Kibana
+  config.log_level = :info                                # Or :warn, or :error
+  config.log_format = :json                               # For parsing in Logit
+  config.rails_semantic_logger.add_file_appender = false  # Don't log to file
+  config.active_record.logger = nil                       # Don't log SQL
+RUBY
+
+insert_into_file(
+  "config/environments/production.rb", 
+  production_config,
```
Comment (peteryates) [inline templates/semantic_logger.rb]: There are a few bits of trailing whitespace here.

**Comment by peteryates** [comment]
Looks great other than the trailing whitespace, happy to merge after that's tweaked. Worked locally for me.

File: templates/semantic_logger.rb (hunk 1 above)
Comment (JamieCleare2525) [reply templates/semantic_logger.rb]: removed in [f5916b1](https://github.com/DFE-Digital/rails-template/pull/32/commits/f5916b12f6335becede4fc14b9081a3687dd0f0b)

**Review by peteryates — APPROVED** [review]

**Review by Kizr — APPROVED** [review]

**Pull request merged**